### PR TITLE
build(infrastructure) connect docker database

### PR DESCRIPTION
**Pull request by marekbaranowski98**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/igroup-opole/yesterday-api/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by marekbaranowski98** [comment]
[[backend] Database Context](https://trello.com/c/FdIbX3sh/24-backend-database-context)

File: YesterdayApi/Startup.cs [hunk 1]
```diff
@@ -23,6 +25,8 @@ public void ConfigureServices(IServiceCollection services)
       services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
       services.AddMapper();
       services.AddSwagger();
+      services.AddDbContext<YesterdayContext>(opt =>
```
Comment (fabiankoziolek) [inline YesterdayApi/Startup.cs]: For better readibility create static function Add(extensionName), the rest looks fine for me

Comment (AlexTorianyk) [inline YesterdayApi/Startup.cs]:
```suggestion
      services.AddDbContext();
```

File: YesterdayApi/appsettings.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "yesterday": "Data Source=127.0.0.1;Initial Catalog=yesterday;persist security info=True;user id=sa;password=zaq1@WSX"
```
Comment (fabiankoziolek) [inline YesterdayApi/appsettings.json]: A better way for the connection string:
"Server=localhost, 1433; Database=DBNAME; MultipleActiveResultSets=true; User Id=LOGIN; Password=PASSWORD;"

**Pull request merged**